src/assets/plugins/filemanager/filemanager.js

```diff
@@ -50,6 +51,33 @@ if (!RedactorPlugins) var RedactorPlugins = {};
 					}, this)
 				});
 
+				var $tabBox3 = $('<div id="redactor-file-manager-delete-box" style="overflow: auto; height: 300px;" class="redactor-tab redactor-tab3">').hide();
```

Comment:
What will happens with this functionality after `Imperavi` redactor will be updated to a new version?